tools-calldata/evm/contracts/RevertConstructor.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
This same test can be found at `Test8.sol` and is generated at `gen-test-deploy.json`.
Maybe we can reuse that test or remove it to avoid duplicity.

Reply:
agree!
I change this PR to draft then until I move the tests